docs/spec.json

```diff
@@ -24,10 +24,12 @@
       "alias": "c",
       "description": "path/to/config.yml where parameters can be set instead of on the command line"
     },
+    "context": {
+      "description": "Specific context that should be used when connecting"
+    },
     "engine": {
       "alias": "e",
-      "description": "URL to the Qlik Associative Engine",
-      "default": "localhost:9076"
```

Comment:
No, I forgot that it was automatically documented, otherwise I would argue that this is a bug and not an feature and it could therefore technically be part of a patch/minor. But, since it is part of the documentation, it would be a breaking change and we should avoid it.

Reply:
Reverted this change.